Hazel/src/Platform/OpenGL/OpenGLRendererAPI.cpp

```diff
@@ -4,9 +4,38 @@
 #include <glad/glad.h>
 
 namespace Hazel {
+	
+#ifdef HZ_ENABLE_RENDERER_LOG
+	void OpenGLRendererAPI::OpenGLMessageCallback(
```

Comment:
This potentially doesn't need to be part of the interface for OpenGLRendererAPI and instead could just a function within this file, this way you don't have to recompile everything if the flag changes, also no need in excluding this function at compile time, the optimizer can take care of that for you

Reply:
I agree, I think this could be a static function that is only defined inside the `OpenGLRendererAPI.cpp` file